jobserver/honeycomb.py

```diff
@@ -130,16 +130,17 @@ def jobrequest_link(job_request):
     url = TemplatedUrl(
         start_time=start,
         end_time=end,
-        breakdowns=["name"],
-        calculations=[{"op": "CONCURRENCY"}],
+        breakdowns=["job", "name"],
+        calculations=[
+            {"op": "CONCURRENCY"},
+            {"op": "MAX", "column": "cpu_percentage"},
+            {"op": "MAX", "column": "memory_used"},
+        ],
         orders=[{"op": "CONCURRENCY", "order": "descending"}],
         stacked=True,
+        omit_missing=True,
```

Comment:
bit: what's the significance of this option? Worth a comment maybe?

Reply:
We're switching trace types. Ticks traces can have missing data, whereas job traces generally do not (as they are historical). So omit_missing values prevents ugly/misleading graphs when that happens.

It's the equivalent of the "Omit missing values" UI option
![image](https://user-images.githubusercontent.com/1042/214307854-88279600-901a-4c58-b933-5424d595adbd.png)